includes/base/element-base.php

```diff
@@ -707,6 +707,527 @@ protected function add_render_attributes() {
 		do_action( 'elementor/element/after_add_attributes', $this );
 	}
 
+	/**
+	 * Register the Transform controls in the advanced tab of the widgets.
+	 *
+	 * Previously registered under the Widget_Common class, but registered a more fundamental level now to enable access from other widgets.
+	 *
+	 * @since 3.7.0
+	 * @access protected
+	 * @return void
+	 */
+	protected function register_transform_section( $container = false ) {
```

Comment:
In order to support future types of elements (such as a planned Grid Container, which will be a separate element type than the Flex Container), use a string parameter instead of a boolean.

In the container's case, this parameter should be `'container'`.

Then, you can use that variable for the various selector/modifier variables, e.g.:
```
protected function register_transform_section( $container = false ) {
...
	$transform_selector_class = ' > .elementor-widget-' . $element_selector;
	$transform_css_modifier = $element_selector . '-';
```
So on and so forth.

Reply:
Other than this - looks good, good work!

Reply:
Thank you, Udi. 

Changes made.